docs/troubleshooting/troubleshooting.mdx

```diff
@@ -25,18 +25,47 @@ description: A guide to troubleshooting SideStore.
      - [Download iTunes](https://www.apple.com/itunes/download/win64/)
 
 
-## Sign-In
+## Sign-In Issues
+
+#### Sidestore Freezing or Displaying an Error Code During Sign-In
+
+If you encounter freezing or error codes while attempting to sign in to Sidestore, there could be several potential causes. Follow these steps to resolve the issue:
+
+1. **Change the Anisette Server:**
+   The most common solution is to switch to a different Anisette server. Anisette servers may occasionally experience downtime. Switching to an alternative server typically resolves the issue. Do this:
+   - Open Sidestore settings.
+   - Scroll down to the "Anisette Server" option.
+   - Select a different server from the list.
+   - We advise you to try the "Macley" server first before the others, as it is one of the most stable.
```

Comment:
No, we are actively working to fix the server problems, but we should not say a specific server to use. Say to use a different server

Reply:
fixed